addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -166,11 +169,30 @@ export class SuggestionService {
                 suggestions.push(partner);
             }
         }
+        specialMentionRegistry
+            .getEntries()
+            .filter(([_name, info]) => {
+                return (
+                    info.channel_types.includes(thread.channel_type) &&
+                    cleanedSearchTerm.length >= Math.min(4, info.label.toString().length) &&
+                    info.label.toString().startsWith(cleanedSearchTerm)
```

Comment:
Ok to keep `Math.min()` to be 100% sure no other special mention in the future could not work at all and we waste time adding this code

Note that `info.label` will no longer be `_t()` so no need to `.toString()`